src/ng/compile.js

```diff
@@ -1386,6 +1386,61 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var ignoreMDirectives = false;
```

Comment:
Either of commentDirectivesDisabled or commentDirectivesEnabled is fine. Just make sure you switch the flag in the latter case ;)